core-services/prow/02_config/openshift/multiarch-tuning-operator/_prowconfig.yaml

```diff
@@ -1,3 +1,12 @@
+branch-protection:
+  orgs:
+    openshift:
+      repos:
+        multiarch-tuning-operator:
+          branches:
+            main:
+              protect: true
```

Comment:
shouldn't we set this to false?

Reply:
That would make scene, but the task said set it to true

Reply:
i assume this is a temporary measure?

Reply:
Ahah my bad then 😅


Yes, we temporarily need to be able to push on main and overcome the failures in Konflux as they block us from merging and we always need to override